src/routes/index.tsx

```diff
@@ -22,6 +23,7 @@ export const Routes = () => (
         <Route path="/revisao/:id" exact component={CordelReview} isPrivate />
         <Route path="/cordeis/:id" exact component={CordelDetails} />
         <Route path="/autores" exact component={AuthorList} isPrivate />
+        <Route path="/autores/:id" exact component={AuthorDetails} isPrivate />

```

Comment:
Essa rota não deveria ser pública como a rota `/cordeis/:id` 
```suggestion
        <Route path="/autores/:id" exact component={AuthorDetails} />

```